unit_tests/testLinearSolver/testLinearSolver.f90

```diff
@@ -1591,20 +1591,20 @@ SUBROUTINE testIterativeOthers()
       CALL thisLS%init(pList)
 
       SELECTTYPE(thisLS); TYPE IS (LinearSolverType_Iterative)
-        CALL thisLS%setConv(-2,-0.1_SRK,-1,-1)
-        CALL thisLS%setConv(-2,1.1_SRK,-1,-1)
+        CALL thisLS%setConv(-2,-0.1_SRK,-0.1_SRK,-1,-1)
+        CALL thisLS%setConv(-2,1.1_SRK,1.1_SRK,-1,-1)
         !Check if default value is used
         bool = thisLS%maxIters == 1000_SIK .AND. thisLS%normType == 2_SIK &
-          .AND. thisLS%convTol == 0.001_SRK .AND. thisLS%nRestart == 30_SIK
+          .AND. thisLS%absConvTol == 0.001_SRK .AND. thisLS%nRestart == 30_SIK
         ASSERT(bool, 'Iterative%setConv(...)')
```

Comment:
Should probably assert that relConvTol got set correctly (and maybe pass in a value different from the absConvTol just to make sure)

Reply:
Already addressed